pennylane/labs/resource_estimation/__init__.py

```diff
@@ -76,6 +76,20 @@
     ~ResourceRZ
     ~ResourceRot
     ~ResourcePhaseShift
+    ~ResourceCH
+    ~ResourceCY
+    ~ResourceCZ
+    ~ResourceCSWAP
+    ~ResourceCCZ
+    ~ResourceCNOT
+    ~ResourceToffoli
+    ~ResourceMultiControlledX
+    ~ResourceCRX
+    ~ResourceCRY
+    ~ResourceCRZ
+    ~ResourceCRot
+    ~ResourceControlledPhaseShift
+    ~ResourceTempAND
```

Comment:
Missing SWAP in docs.

```suggestion
    ~ResourceSWAP
    ~ResourceCH
    ~ResourceCY
    ~ResourceCZ
    ~ResourceCSWAP
    ~ResourceCCZ
    ~ResourceCNOT
    ~ResourceToffoli
    ~ResourceMultiControlledX
    ~ResourceCRX
    ~ResourceCRY
    ~ResourceCRZ
    ~ResourceCRot
    ~ResourceControlledPhaseShift
    ~ResourceTempAND
```